redwind/contexts.py

```diff
@@ -46,20 +46,20 @@ def extract_ogp_context(context,doc,url):
     ogp_url = soup.find('meta',{'property':'og:url'})
     ogp_content = soup.find('meta',{'property':'og:description'})
 
-    if ogp_title:
+    if ogp_title and not context.title:
         context.title = ogp_title['content']
```

Comment:
will BS4 throw an KeyError here if the meta tag doesn't have a content property? if so it might be better to use .get('content')